docs/doc/60-contributing/03-rfcs/20220809-share.md

```diff
@@ -0,0 +1,210 @@
+---
+title: Share objects
+description: Share objects between tenants
```

Comment:
Please define tenant first.

Reply:
The document preview address is: https://databend-git-fork-lichuang-addsharerfc-databend.vercel.app/doc/contributing/rfcs/share